.github/workflows/add-issues-to-project.yaml

```diff
@@ -1,18 +1,20 @@
-name: Auto Assign to Project(s)
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# From https://github.com/github/feedback/discussions/5378#discussioncomment-2137318
+name: Project automation
 
 on:
   issues:
-    types: [opened]
-env:
-  MY_GITHUB_TOKEN: ${{ secrets.MY_GITHUB_TOKEN }}
+    types:
+      - opened
 
 jobs:
-  assign_one_project:
+  add_issue_to_project:
+    name: Add new issues to project board
     runs-on: ubuntu-latest
-    name: Assign to One Project
     steps:
-    - name: Assign NEW issues to project
-      uses: srggrs/assign-one-project-github-action@1.3.1
-      if: github.event.action == 'opened'
-      with:
-        project: 'https://github.com/orgs/kubeapps/projects/4'
```

Comment:
Could the reason this was failing be related to it being for the wrong project? (haven't checked, but this is the old project under the kubeapps org)

Reply:
No, I checked it at some point, but this action only works with standard projects, not the beta ones :(